src/main/java/com/glodblock/github/proxy/CommonProxy.java

```diff
@@ -68,8 +66,6 @@ public void postInit(FMLPostInitializationEvent event) {
         }
         AEApi.instance().registries().externalStorage().addExternalStorageInterface(new AEFluidInterfaceHandler());
         InterfaceTerminalSupportedClassProvider.register(PartFluidP2PInterface.class);
-        InterfaceTerminalSupportedClassProvider.register(PartFluidInterface.class);
-        InterfaceTerminalSupportedClassProvider.register(TileFluidInterface.class);
```

Comment:
this is fine, but also needs to remove the ASM in these places:
1. https://github.com/GTNewHorizons/AE2FluidCraft-Rework/blob/master/src/main/java/com/glodblock/github/coremod/transform/ContainerInterfaceTerminalTransformer.java
2. https://github.com/GTNewHorizons/AE2FluidCraft-Rework/blob/master/src/main/java/com/glodblock/github/coremod/FCClassTransformer.java#L20